src/main/java/laststep/domain/Cars.java

```diff
@@ -0,0 +1,69 @@
+package laststep.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import laststep.service.MoveStrategy;
+import laststep.service.RandomMoveStrategy;
+
+public class Cars {
+    private final MoveStrategy strategy;
```

Comment:
생성자에서 초기화하지 않고 선언과 동시에 초기화해도 괜찮을 것 같아요 😃 
```suggestion
    private final MoveStrategy strategy = new RandomMoveStrategy();;
```